.gitignore

```diff
@@ -2,4 +2,8 @@
 /dist
 /lib
 *.pem
+<<<<<<< Updated upstream
```

Comment:
Is this supposed to be here? Looks like merge conflict/resolution lines from git.